draft-ietf-taps-impl.md

```diff
@@ -135,7 +135,9 @@ Listener objects are created with a Preconnection, at which point their configur
 
 # Implementing Pre-Establishment
 
-During pre-establishment the application specifies the Endpoints to be used for communication as well as its preferences regarding Protocol and Path Selection. The implementation stores these objects and properties as part of the Preconnection object for use during connection establishment. For Protocol and Path Selection Properties that are not provided by the application, the implementation must use the default values specified in the Transport Services API ({{I-D.ietf-taps-interface}}).
+During pre-establishment the application specifies the Endpoints to be used for communication as well as its preferences via Selection Properties and, if desired, also Connection Properties. Generally, Connection Properties should be configured as early as possible, as they may serve as input to decisions that are made by the implementation (the Capacity Profile may guide usage of a protocol offering scavenger-type congestion control, for example). In the remainder of this document, we only refer to Selection Properties because they are the more typical case.
```

Comment:
Perhaps we should add to the last sentence "we only refer to Selection Properties because they are the more typical case and have to be handled by all implementations" or similar.  

An implementation could ignore the Connection properties, but not the Selection properties (it then depend on their setting if they can be ignored in the second step or not, but they have to be considered and if set to require or prohibit also satisfied).

Reply:
Done with the wording you propose